kpi/views.py

```diff
@@ -169,13 +169,6 @@ def perform_create(self, serializer):
             serializer.save()
 
     def perform_destroy(self, instance):
-        # Only directly-applied permissions may be modified; forbid deleting
```

Comment:
Some permissions marked as inherited really can't be removed, like the permissions assigned to the owner. I'm going to play around and see how `remove_perm()` (on line 179 below) handles those cases.